app/assets/stylesheets/dreamsis.css.erb

```diff
@@ -2316,6 +2321,72 @@ body.attendance .success-marker {
   background-color: rgba(235, 240, 249, 0.25);
 }
 
+.attendance .optioned .choice-dropdown {
```

Comment:
Please feel free to tear apart my CSS style, it's been years since I've written CSS this complex